action.yml

```diff
@@ -1,15 +1,148 @@
-name: 'Rainforest QA: RFML Runner'
-description: 'Run RFML tests from your repo'
+name: Rainforest QA GitHub Action
+description: Easily trigger a Rainforest Run from your GitHub workflows
 inputs:
-  rainforest_api_key:  # id of input
-    description: 'Rainforest API key'
+  description:
+    description: An arbitrary string to associate with the run
+    required: false
+    default: ''
+
+  run_group_id:
+    description: Only run tests tied to this Run Group
     required: true
+
+  environment_id:
+    description: Use a specific environment for this run
+    required: false
+    default: ''
+
+  conflict:
+    description: How other currently in-progress runs should be handled. Values are `abort` to cancel runs in the same environment as your new run and `abort-all` to cancel all runs
+    required: false
+    default: ''
+
+  crowd:
+    description: The crowd to use for this run
+    required: false
+    default: ''
+
+  release:
+    description: Manually entered release information about the release the run is associated with
+    required: false
+    default: ''
+
+  token:
+    description: Your Rainforest QA API token
+    required: true
+
+  dry_run:
+    description: Set to true to run parameter validations without starting a new Rainforest run
+    required: false
+    default: ''
+
+  background:
+    description: Do not wait for a run to complete before exiting
+    required: false
+    default: ''
 runs:
-  using: 'docker'
-  image: 'docker://gcr.io/rf-public-images/rainforest-cli:latest'
-  args:
-    - 'run'
-    - '--fail-fast'
-    - '-f'
-    - 'spec/rainforest'
+  using: composite
+  steps:
+    - name: Set Action Version
+      shell: bash
+      run: |
+        echo "version=0.0.1" >> $GITHUB_ENV
+    - name: Validate Parameters
+      shell: bash
+      id: validate
+      run: |
```

Comment:
Might already be the case that GH invoke bash with extra args? If not, make bash fail on errors:

```suggestion
      run: |
        set +euo pipefail
```

Reply:
This is how GitHun runs `bash`: `bash --noprofile --norc -eo pipefail {0}`
https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsshell